app/src/main/java/com/github/se/orator/ui/navigation/NavigationActions.kt

```diff
@@ -51,6 +53,9 @@ object Screen {
   const val FEEDBACK = "Feedback Screen"
   const val CHAT_SCREEN = "chat_screen"
   const val SPEAKING = "Speaking"
+  const val PRACTICE_QUESTIONS_SCREEN = "PracticeQuestions Screen"
```

Comment:
Is there a difference between `PRACTICE_QUESTIONS_SCREEN` and the unused constant `PRACTICE_SCREEN` ?

Reply:
oops- missed that